### PR TITLE
Remove Weakmap.clear()

### PR DESCRIPTION
**Pull request by teoli2003**
Fixes ##20040

### CODE REVIEW

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -12209,6 +12209,7 @@
 /en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/TypedArray.prototype.reduce()	/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/reduce
 /en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/prototype	/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray
 /en-US/docs/Web/JavaScript/Reference/Global_Objects/URIError/prototype	/en-US/docs/Web/JavaScript/Reference/Global_Objects/URIError
+/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap/clear	/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap
```
Comment (Josh-Cena) [inline files/en-us/_redirects.txt:12212]: Redirect to the obsolete features page and add this one there? (I like that page; I like trivia)

Comment (teoli2003) [reply files/en-us/_redirects.txt:12212]: Not sure to understand…

Comment (Josh-Cena) [reply files/en-us/_redirects.txt:12212]: Add it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Deprecated_and_obsolete_features

We didn't mention `clear()` at all on the weak map page, so redirecting it there is disorienting.

Comment (teoli2003) [reply files/en-us/_redirects.txt:12212]: We always do this.

Comment (Josh-Cena) [reply files/en-us/_redirects.txt:12212]: TBH, I always feel like it's a form of "cyber-gaslighting" when I click on a link and it takes me to redirect a page that has no mention of what I intend to see. It's like "it's all an illusion; there no such thing as `WeakMap.prototype.clear()`"... I also intend to keep the deprecated/obsolete features list more well-maintained in the future.

Comment (Elchi3) [reply files/en-us/_redirects.txt:12212]: Looks like I just removed the page for WeakSet.clear without any redirect. https://github.com/mdn/content/pull/4671
So there isn't an (enforced) standardized way to do this.

I like the idea to redirect to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Deprecated_and_obsolete_features but that indeed requires us to maintain that page.

Comment (Josh-Cena) [reply files/en-us/_redirects.txt:12212]: I had sent a few PRs in the past to update that page and I think at this point it's in pretty good shape to be redirected to. (And I already redirect the array comprehension pages to that)

Comment (wbamberg) [reply files/en-us/_redirects.txt:12212]: > TBH, I always feel like it's a form of "cyber-gaslighting" when I click on a link and it takes me to a page with no mention of what I intend to see. It's like "it's all an illusion; there no such thing as `WeakMap.prototype.clear()`"... I also intend to keep the deprecated/obsolete features list more well-maintained in the future.

Yeah, I think this is a good point.

But we would need a practice of maintaining the obsolete features list - and not just in JS, if we wanted to make this a general practice.

Comment (Josh-Cena) [reply files/en-us/_redirects.txt:12212]: AFAICT, web APIs are deprecated much more aggressively and widely than JS APIs, so I'm not sure how well that scales. It definitely comes to stability in JS now, and probably won't update more than once every few years.

**Pull request closed without merging**